core/templates/vector.h

```diff
@@ -69,7 +69,12 @@ class Vector {
 	bool push_back(T p_elem);
 	_FORCE_INLINE_ bool append(const T &p_elem) { return push_back(p_elem); } //alias
 	void fill(T p_elem);
-
+	T pop_back() {
```

Comment:
```suggestion

	T pop_back() {
```
Keep an empty line here for readability